examples/feature_demo/scene_subplots_video.py

```diff
@@ -85,15 +85,15 @@ def animate():
         img.geometry.grid.update_full()
 
     # render the viewports
-    for viewport, s, c in zip(viewports, scenes, cameras):
+    for viewport, s, c in zip(viewports, scenes, cameras, strict=True):
```

Comment:
i feel like "strict=True" is the correct way to zip is that right?

Reply:
I agree that it's the more correct way. I wish it was the default. Since it's also more verbose I personally don't care in cases were you know both iterables are the same length ... but I'm fine with these changes 😉

Reply:
Just fyi. Ruff complained without this